SwrveSDK/SDK/Track/Swrve.h

```diff
@@ -2,6 +2,7 @@
 #import "SwrveInterfaceOrientation.h"
 #import "SwrveReceiptProvider.h"
 #import "SwrveResourceManager.h"
+#import "SwrveBaseCampaign.h"
```

Comment:
There's a reason why this is required here, right? Might be worth adding comment in case we accidentally remove it at a later date?